File: UserServer/src/main/java/cc/nobrain/dev/userserver/domain/base/controller/GlobalController.java
package cc.nobrain.dev.userserver.domain.base.controller;

import cc.nobrain.dev.userserver.common.component.FileComponent;
import lombok.RequiredArgsConstructor;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import org.springframework.web.multipart.MultipartFile;

import java.net.http.HttpResponse;

@RestController
@RequiredArgsConstructor
@RequestMapping("/api")
public class GlobalController {

    private final FileComponent fileComponent;

    @GetMapping("/health")
    public ResponseEntity health() {
        return ResponseEntity.ok().build();
    }
}

File: UserServer/src/main/java/cc/nobrain/dev/userserver/domain/base/controller/GlobalController.kt
package cc.nobrain.dev.userserver.domain.base.controller

import cc.nobrain.dev.userserver.common.component.FileComponent
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api")
class GlobalController(private val fileComponent: FileComponent) {

    @GetMapping("/health")
    fun health(): ResponseEntity<Unit> {
        return ResponseEntity.ok().build()
    }
}